html/changelog.html

```diff
@@ -1286,33 +1286,6 @@ <h3 class="author">grinkgo updated:</h3>
 			<ul class="changes bgimages16">
 				<li class="bugfix">sulaco medbay lobby entrance isn't id-locked anymore</li>
 			</ul>
-
-			<h2 class="date">27 January 2021</h2>
-			<h3 class="author">Pariah919 updated:</h3>
-			<ul class="changes bgimages16">
-				<li class="balance">healing infusion heals a flat 25 hp instead of 10+5% and heals less sunder</li>
-			</ul>
-			<h3 class="author">SpaceLove updated:</h3>
-			<ul class="changes bgimages16">
-				<li class="rscadd">Synthetic ranks added</li>
-			</ul>
-			<h3 class="author">Sulaboy updated:</h3>
-			<ul class="changes bgimages16">
-				<li class="bugfix">M412L1 and M412 Elite now show correct stock</li>
-				<li class="bugfix">M412L1 can now attach rail-scope, and mini-scope</li>
-			</ul>
-			<h3 class="author">Surrealaser updated:</h3>
-			<ul class="changes bgimages16">
-				<li class="bugfix">Fixes damage delays for most relevant damage events.</li>
-			</ul>
-			<h3 class="author">TiviPlus updated:</h3>
-			<ul class="changes bgimages16">
-				<li class="refactor">Adds the managed spawner controller</li>
-			</ul>
-			<h3 class="author">grinkgo updated:</h3>
-			<ul class="changes bgimages16">
-				<li class="bugfix">Sulaco's medbay has a brand new layout.</li>
-			</ul>
```

Comment:
the fuck is that

Reply:
more stuff that got caught in me hastily slapping this PR together, give me just a few moments.